docs/source/release/v6.4.0/Muon/MA_FDA/Bugfixes/33890.rst

```diff
@@ -0,0 +1,2 @@
+- The Muon Analysis and Frequency Domain Analysis GUI's have had a bug fixed that caused a crash when changing instrument.
+ - The Muon Analysis and Frequency Domain Analysis GUI's have had a bug fixed that caused a crash when undoing a fit.
```

Comment:
Is this line supposed to have an extra space indent? If not, rst might do something weird with it!